.github/workflows/test-openmcyclus.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           mamba env update -n openmcyclus-env -f environment.yml
 
       - name: Install pytest
-        run: conda install -y pytest nose
+        run: conda install -y pytest
```

Comment:
This works, but I remembered doing something slightly different when installing pytest. The [anaconda site](https://anaconda.org/anaconda/pytest) recommends `conda install -c anaconda pytest`. I'm sure the flag doesn't matter, but do you know if the anaconda does anything else? (this isn't an issue, I'm just curious if there's an edge case this might miss because they recommend something slightly different)

Reply:
The `-c` flag specifies the channel to get the package from. I believe that for this environment I use conda-forge as the initial one to look in, but if it's not in conda-forge then conda looks in the anaconda channel.